### ISSUE TITLE
does tiedot has max storage limit?

### ISSUE THREAD

**Issue by karterlizard** [HouzuoGuo/tiedot#170]
It works well when I check it yesterday, it has stores 51GB data.
today it grows up to 61GB, it return nothing by http api, also by golang script.

**Comment by HouzuoGuo** [comment]
You're right, I overlooked the maximum size limit and really should have put it in a wiki page somewhere - sorry about that. I wish I had a workaround for this, but cannot think of any right now.
